homeassistant/components/smartenergy_awattar/state.py

```diff
@@ -0,0 +1,11 @@
+"""Awattar state management."""
+
+from awattar_api.awattar_api import AwattarApi
+
+from .const import API, UNSUB_OPTIONS_UPDATE_LISTENER
+
+
+def init_state(url: str) -> dict:
```

Comment:
Can you please describe the reason for this function/file? Why you don't do it directly in the `__init__.py`